{{ cookiecutter.project_shortname }}/run-tests.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
`/bin/sh` is even more universal, we shouldn't really have to require `bash` shell?